src/docs/topics/red-hat-ecosystem-catalog-certification/README.md

```diff
@@ -0,0 +1,34 @@
+# Managing the Orchard Core Red Hat Ecosystem Catalog certification
+
+The [Red Hat Ecosystem Catalog](https://catalog.redhat.com/) is a catalog of software certified to work on the Red Hat platforms, like directly on Red Hat Enterprise Linux (RHEL). Orchard Core is cross-platform and thus works on RHEL as well. To showcase this, and to publicize Orchard Core better to the Red Hat community, we have certified Orchard and created [its Red Hat Ecosystem Catalog profile](https://catalog.redhat.com/software/applications/detail/223797).
+
+## Renewing the certification
+
+The certification is an assurance that Orchard Core works on RHEL. To keep this certification up-to-date, we need to rerun certification for each new Orchard release, as well as RHEL major release. This is not mandatory though, and it's good enough if we do this certification occasionally.
```

Comment:
We could add this step when we publish a new release to keep it up-to-date

Reply:
Hope we don't forgot this note :)

Reply:
I'm not done yet :).